### ISSUE TITLE
Anchore import breaks

### ISSUE THREAD

**Issue by madchap** [DefectDojo/django-DefectDojo#1069]
**Bug description**
Server internal error 500 on importing files that containe severities "Negligible".

**Steps to reproduce**
Import the following file, dev branch.
See it bomb.

[anchore_v0.3.3_openjdk-11.0.3-jdk-stretch.json.txt](https://github.com/DefectDojo/django-DefectDojo/files/3093608/anchore_v0.3.3_openjdk-11.0.3-jdk-stretch.json.txt)

**Expected behavior**
The import goes through.

**Deployment method** *(select with an `X`)*
- [ ] Kubernetes
- [x] Docker
- [ ] setup.bash / legacy-setup.bash

**Environment information**
- dev branch, docker.

**Console logs** (optional)
```
uwsgi_1         | Internal Server Error: /product/1/import_scan_results
uwsgi_1         | Traceback (most recent call last):
uwsgi_1         |   File "/usr/local/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
uwsgi_1         |     response = get_response(request)
uwsgi_1         |   File "/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
uwsgi_1         |     response = self._get_response(request)
uwsgi_1         |   File "/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
uwsgi_1         |     response = self.process_exception_by_middleware(e, request)
uwsgi_1         |   File "/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
uwsgi_1         |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
uwsgi_1         |   File "/usr/local/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
uwsgi_1         |     return view_func(request, *args, **kwargs)
uwsgi_1         |   File "/app/dojo/product/views.py", line 382, in import_scan_results_prod
uwsgi_1         |     return import_scan_results(request, pid=pid)
uwsgi_1         |   File "/usr/local/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
uwsgi_1         |     return view_func(request, *args, **kwargs)
uwsgi_1         |   File "/app/dojo/engagement/views.py", line 531, in import_scan_results
uwsgi_1         |     if Finding.SEVERITIES[sev] > Finding.SEVERITIES[min_sev]:
uwsgi_1         | KeyError: u'Negligible'
uwsgi_1         | Internal Server Error: /product/1/import_scan_results
uwsgi_1         | Traceback (most recent call last):
uwsgi_1         |   File "/usr/local/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
uwsgi_1         |     response = get_response(request)
uwsgi_1         |   File "/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
uwsgi_1         |     response = self._get_response(request)
uwsgi_1         |   File "/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
uwsgi_1         |     response = self.process_exception_by_middleware(e, request)
uwsgi_1         |   File "/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
uwsgi_1         |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
uwsgi_1         |   File "/usr/local/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
uwsgi_1         |     return view_func(request, *args, **kwargs)
uwsgi_1         |   File "/app/dojo/product/views.py", line 382, in import_scan_results_prod
uwsgi_1         |     return import_scan_results(request, pid=pid)
uwsgi_1         |   File "/usr/local/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
uwsgi_1         |     return view_func(request, *args, **kwargs)
uwsgi_1         |   File "/app/dojo/engagement/views.py", line 531, in import_scan_results
uwsgi_1         |     if Finding.SEVERITIES[sev] > Finding.SEVERITIES[min_sev]:
uwsgi_1         | KeyError: u'Negligible'
```

**Comment by madchap** [comment]
The easiest way is probably to mark these "Negligible" into "Info" to not change the model. The semantic sounds the same to me.

**Comment by madchap** [comment]
Same error for `Unknown`. Will fix the same way for now.

**Comment by valentijnscholten** [comment]
fixed by #1070
